park-dashboard/src/Components/WeatherForecast/WeatherForecast.js

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useEffect, useRef } from "react";
+import moment from "moment";
+import "./WeatherForecast.css";
+import { WiCloudy, WiRain, WiSnow } from "react-icons/wi"; // Import weather icons from react-icons library
```

Comment:
Can you consolidate both the imports together as they are from the same library? Also WiCloud seems to be not used. Can you remove it if it's not being used?

Reply:
Consolidated the import sentences.